libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -86,15 +81,38 @@ export class UserProfileService {
       nationalId: user.nationalId,
       //temporary as schemas where not working properly
       locale: input.locale as string,
-      mobilePhoneNumber: input.mobilePhoneNumber,
-      email: input.email,
+
+      /* Mobile and email will be within islykill service */
+      // mobilePhoneNumber: input.mobilePhoneNumber,
+      // email: input.email,
     }
     const request: UserProfileControllerCreateRequest = {
       createUserProfileDto: createUserDto,
     }
-    return await this.userProfileApiWithAuth(user)
+
+    const feature = await this.featureFlagService.getValue(
+      Features.personalInformation,
+      false,
+      user,
+    )
+
+    if (!feature) {
+      handleError({ status: 'User profile create is feature flagged for user' })
```

Comment:
Should not throw an error? If the feature is disabled we still want to update the locale.

Reply:
Yeah, whoops, this should definitely not throw an error.